src/components/v5/common/ActionSidebar/partials/forms/EditTeamForm/utils.tsx

```diff
@@ -7,17 +10,32 @@ export const getEditDomainPayload = (
   colony: Colony,
   values: EditTeamFormValues,
   selectedDomain: Domain,
-) => ({
-  colonyAddress: colony.colonyAddress,
-  colonyName: colony.name,
-  domainName: values.teamName,
-  domainColor: values.domainColor,
-  domainPurpose: values.domainPurpose,
-  domain: selectedDomain,
-  annotationMessage: values.description
-    ? sanitizeHTML(values.description)
-    : undefined,
-  customActionTitle: values.title,
-  motionDomainId: values.createdIn,
-  isCreateDomain: false,
-});
+) => {
+  const baseDomainPayload = {
+    colonyAddress: colony.colonyAddress,
+    colonyName: colony.name,
+    domainName: values.teamName,
+    domainColor: values.domainColor,
+    domainPurpose: values.domainPurpose,
+    domain: selectedDomain,
+    annotationMessage: values.description
+      ? sanitizeHTML(values.description)
+      : undefined,
+    customActionTitle: values.title,
+    isCreateDomain: false,
+  };
+
+  if (values.decisionMethod === DecisionMethod.Permissions) {
+    return {
+      ...baseDomainPayload,
```

Comment:
we can not spread and just directly return the value

Reply:
Ah yes you're right. This used to have other fields apart from the `baseDomainPayload` which I removed last minute